dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -1200,34 +1170,32 @@ can manage multilingual inserts for already stored records but not for the case
                     }
                 }
 
+                //Set the site and folder
+                if (siteAndFolder != null) {
+                    final Host host = siteAndFolder.getLeft();
+                    final Folder folder = siteAndFolder.getRight();
+                    if (UtilMethods.isSet(folder) && !folder.isSystemFolder()) {
+                        if (!permissionAPI.doesUserHavePermission(folder,PermissionAPI.PERMISSION_CAN_ADD_CHILDREN,user)) {
+                            throw new DotSecurityException( "User has no Add Children Permissions on selected folder" );
+                        }
+                    } else if (UtilMethods.isSet(host)) {
+                        if (!permissionAPI.doesUserHavePermission(host,PermissionAPI.PERMISSION_CAN_ADD_CHILDREN,user)) {
+                            throw new DotSecurityException("User has no Add Children Permissions on selected host");
```

Comment:
I think we use site instead of host, but not a big deal

Reply:
Changed variable name to site